### DISCUSSION TITLE
How to scale down / zoom out / reduce the size of the draw io service menu / UI - it it got huge and I see no way to make it smaller see screenshot

### DISCUSSION THREAD

**Discussion by art-ip** [jgraph/drawio-desktop#1376] [Q&A]
How to scale down / zoom out / reduce the size of the draw io service menu / UI - it it got huge and I see no way to make it smaller see screenshot 
![image](https://github.com/jgraph/drawio-desktop/assets/130511582/2c13b638-9d9c-4c0f-9811-9cdc84aa546f)

**Comment by davidjgraph** [comment]
You can zoomed in using the browser controls, you need to zoom out using the browser controls.

**Reply by davidjgraph** [reply]
Oh, desktop, ok, what's the version?

**Reply by davidjgraph** [reply]
You've got bookmarks in there, how can this be desktop?

**Reply by didim99** [reply]
@davidjgraph hello, I have a similar (but opposite) question, draw.io desktop 27.0.5 (Ubuntu deb package), how to increase UI scaling in application? Can't find any ralated option is Settings or app main menu. :(

**Comment by art-ip** [comment]
I found the solution. Activate Number Pad (by pressing NumLock) and then
press Crt+0 on the numeric pad.
